content/v2.0/influxdb-templates/stacks/_index.md

```diff
@@ -13,38 +11,104 @@ related:
   - /v2.0/reference/cli/influx/pkg/stack/
 ---
 
-**InfluxDB stacks** are **stateful [InfluxDB templates](/v2.0/influxdb-templates)**
-that let you add, update, and remove templated resources over time, avoid
-duplicating resources when applying the same or similar templates more than once,
-and apply changes to distributed instances of InfluxDB OSS or InfluxDB Cloud.
+**InfluxDB stacks** are stateful [templates](/v2.0/influxdb-templates) that let you apply changes
+to distributed instances of InfluxDB OSS or InfluxDB Cloud. Stacks (like templates) are groups of
+pre-defined resources, including buckets, dashboards, tasks, checks, labels, and Telegraf configurations.
 
-## Ideal use cases for InfluxDB stacks
-Stacks help save time and effort in the following use cases:
+- [Review ideal use cases for InfluxDB stacks](#review-ideal-use-cases-for-influxdb-stacks),
+  like automating InfluxDB deployments with GitOps and stacks.
+- [Manage InfluxDB stacks](#manage-influxdb-stacks)
 
-- [Actively develop and extend templates](#actively-develop-and-extend-templates)
+{{% note %}}
+**Key differences between stacks and templates**:
+
+- A template is a file outside of InfluxDB. Once a template is applied to InfluxDB, it becomes a stack.
```

Comment:
When you apply an InfluxDB template, it automatically creates a stack which allows you to manage the state of that template. I'm not sure if "becomes a stack" is the right language here.

Reply:
Thank you, Russ!  Re what it means to "manage the state of a template:"

- If you apply a stack, it knows your environment has a resource. 
- If you apply a template, it doesn't know your environment has a resource. 

To help folks understand, perhaps we can clarify the reason why we made templates not recognize resources. What is the use case?

Reply:
yeah, a stack has a mapping between the meta.name field of the template file and the id of the created resource. When you "apply" the template again after making changes, it will manipulate the existing resource as long as you keep the meta.name field the same in the file/url/stream.

The reason for the two different "things" (template and stack) is that there are cases where you would want to duplicate a resource (maybe i want to bring in another copy of a dashboard or task and make changes to it) and there are instances where you would want to update an existing resource (deploying changes from a github repo to an environment).